### PR TITLE
no output type: null -> none

### PR DESCRIPTION
**Pull request by wilzbach**
`null` comes from JSON, but the specification is in YAML.
Hence, `none` is a more natural fit.

### CODE REVIEW

**Comment by judepereira** [comment]
@wilzbach I like the null - since it's easily understood, and already used in several languages. Need somebody else to weigh in here too. @microservices/storyscript-core - thoughts?

**Comment by wilzbach** [comment]
copy/pasting arguments from Slack:

- We agreed that `null` is a weird word and that we ideally we don't want to use it for Storyscript. Also this is the only time null is every used in OMG spec
- We don't want to minimize words which require explanations / prior CS knowledge.
- For OMG it makes no sense as (3) OMG is supposed to be protocol-agnostic and there's simply no null on the CLI nor is there for e.g. Protobuf.

tl;dr: there's no need to introduce `null` concepts into OMG for this sole occurrence.

I don't mind if another word like `no`, `empty` or `void` is used, I just think that `null` isn't a great fit here.
